vmdb/config/vmdb.tmpl.yml

```diff
@@ -284,6 +284,7 @@ workers:
           :cinder: "notifications.*"
           :glance: "notifications.*"
           :quantum: "notifications.*"
+          :ironic: "conductor_manager.*"
```

Comment:
Actually I should pull this part out of the PR. This was part of my attempt to start seeing Host events for the OpenStack infra provider after talking with @blomquisg about it. However, I haven't proven that this is exactly what I need, so it belongs to the follow-up PR with verified host events. The other files in this PR are cleanup bits that were problematic with the initial change that added this stuff. I'll remove this file and update the PR.